text/reporting.tex

```diff
@@ -402,7 +402,17 @@ \section{Aspect 3: Subsystems Included in Instrumented Power}
 
 
 \noindent
-For Level 1, only include the compute-node subsystem. For some systems, it may be impossible not to include a power contribution from some subsystems. In this case, provide a list of the measured subsystems, but do not subtract an estimated value for the included subsystem. 
+For Level 1, the only subsystem that requires an actual power measurement is the compute-node subsystem. 
```

Comment:
Re-word to note that both network and compute need to be reported, but only the compute needs to be measured.